frontend/www/js/omegaup/contest/edit.ts

```diff
@@ -65,12 +65,29 @@ OmegaUp.on('ready', () => {
           })
           .catch(ui.apiError);
       },
-      refreshProblems: (): void => {
+      refreshProblems: (problemAdded: boolean): void => {
         api.Contest.problems({
           contest_alias: payload.details.alias,
         })
           .then((response) => {
             contestEdit.problems = response.problems;
+            if (problemAdded) {
+              if (
```

Comment:
```suggestion
            if (problemAdded &&
```

así no hay necesidad de tener dos `if` anidados.